rclpy/test/test_node.py

```diff
@@ -176,6 +176,12 @@ def test_service_names_and_types(self):
         # test that it doesn't raise
         self.node.get_service_names_and_types()
 
+    def test_client_names_and_types_by_node(self):
+        # test that it doesnt raise
+        # node1 = rclpy.create_node(
+        #         'my_node', namespace='/my_ns', use_global_arguments=True, context=context)
+        self.node.get_client_names_and_types_by_node(TEST_NODE, TEST_NAMESPACE)
```

Comment:
I thought the service variant was already being tested, perhaps we add a similar smoke test for `get_service_names_and_types_by_node` too.